### DISCUSSION TITLE
Wayland: don't grab "Super" key

### DISCUSSION THREAD

**Discussion by manio** [FreeRDP/FreeRDP#6768] [Q&A] [answered]
Hi!
Some time ago I was trying wlfreerdp in sway/wayland but it was not usable. Today I just gave it another try (only changing xfreerdp to wlfreerdp) and it seems it is a huge step forward. It seems it is usable now. This is great! Thank you for your work on this! :)

SwayWM by default is using Super/Win key as a modifier:
```set $mod Mod4```

My problem is that using the old xfreerdp the "Super" key was not grabbed by xfreerdp client so I was able to do all swaywm stuff like moving a xfree window, etc.
Currently the "Super" key is also passed to the RDP Client. Is there some way to not pass it (preserve xfreerdp behavior), so I can still use it in sway?

**Comment by akallabeth** [comment, marked as answer]
currently we only support keyboard grabbing on or off with `wlfreerdp`, so no exceptions for any keys.

**Reply by manio** [reply]
Oh... thanks :)
it seems that ```-grab-keyboard``` parameter just do the job :)

**Reply by bob333** [reply]
Also use "-f" for fullscreen. Making full screen in sway with key "mod+f" does not scale the contents of the screen.
